stl/inc/xpolymorphic_allocator.h

```diff
@@ -269,8 +271,15 @@ namespace pmr {
         template <class _Uty, class... _Types>
         void construct(_Uty* const _Ptr, _Types&&... _Args) {
             // propagate allocator *this if uses_allocator_v<_Uty, polymorphic_allocator>
+#if _HAS_CXX20
+            _STD apply(
+                [_Ptr](
+                    auto&&... _New_args) { _STD construct_at(_Ptr, _STD forward<decltype(_New_args)>(_New_args)...); },
+                uses_allocator_construction_args<_Uty>(*this, _STD forward<_Types>(_Args)...));

```

Comment:
The formatting here is a bit off.

Reply:
Well, that's what clang-format wants...

Reply:
When clang-format wraps in an unexpected manner, it's sometimes possible to encourage nicer output (without taking the extreme step of disabling it entirely) by adding empty comments: `//` at the end of a line.

Reply:
I ended up using `uninitialized_construct_using_allocator` here.